app/src/main/java/com/example/pictures/ui/viewmodel/PicturesViewModel.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.pictures.ui.viewmodel
+
+import android.util.Log
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.pictures.data.models.Photo
+import com.example.pictures.domain.repo.PictureRepo
+import com.example.pictures.domain.repo.PictureRepoImpl
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+class PicturesViewModel : ViewModel() {
+    private var pictureRepo: PictureRepo = PictureRepoImpl()
+
+    private val _isLoading = MutableLiveData<Boolean>().apply { value = false }
+    val isLoading: LiveData<Boolean> = _isLoading
+
+    private val _status = MutableLiveData("")
+    val status: LiveData<String> = _status
+
+    private val _photo = MutableLiveData<List<Photo>>()
+    val photo: LiveData<List<Photo>> = _photo
+
+    init {
+        getDataFromInternet()
+    }
+
+    private fun getDataFromInternet() {
+        Log.d("mylog", "запустилась getDataFromInternet()")
+        viewModelScope.launch {
+            _isLoading.value = true
+            try {
+                val response = withContext(Dispatchers.IO) {
+                    Log.d("mylog", "ретрофит")
```

Comment:
Comments should be written in english, `TAG` should not be abstract and should help to understand where this log was captured. It is better to use `Timber`